python/lsst/meas/algorithms/secondMomentStarSelector.py

```diff
@@ -388,34 +388,18 @@ def getClumps(self, tableVersion, sigma=1.0, display=False):
         # Sources, it's only used to characterize this PSF histogram
         #
         schema = afwTable.SourceTable.makeMinimalSchema()
-        schema.setVersion(tableVersion)
-        if tableVersion == 0:
-            psfImageConfig = SourceMeasurementConfig()
-            psfImageConfig.slots.centroid = "centroid.sdss"
-            psfImageConfig.slots.psfFlux = "flux.psf"
-            psfImageConfig.slots.apFlux = "flux.naive"
-            psfImageConfig.slots.modelFlux = None
-            psfImageConfig.slots.instFlux = None
-            psfImageConfig.slots.shape = "shape.sdss"
-            psfImageConfig.algorithms.names = ["flags.pixel", "shape.sdss",
-                                                           "flux.psf", "flux.naive"]
-            psfImageConfig.centroider.name = "centroid.sdss"
-            psfImageConfig.algorithms["flux.naive"].radius = 3.0
-            psfImageConfig.validate()
-            task = SourceMeasurementTask(schema, config=psfImageConfig)
-        else:
-            psfImageConfig = SingleFrameMeasurementConfig()
-            psfImageConfig.slots.centroid = "base_SdssCentroid"
-            psfImageConfig.slots.psfFlux = "base_PsfFlux"
-            psfImageConfig.slots.apFlux = "base_NaiveFlux"
-            psfImageConfig.slots.modelFlux = None
-            psfImageConfig.slots.instFlux = None
-            psfImageConfig.slots.shape = "base_SdssShape"
-            psfImageConfig.algorithms.names = ["base_SdssCentroid", "base_PsfFlux", "base_NaiveFlux",
-                                               "base_PixelFlags", "base_SdssShape"]
-            psfImageConfig.algorithms["base_NaiveFlux"].radius = 3.0
-            psfImageConfig.validate()
-            task = SingleFrameMeasurementTask(schema, config=psfImageConfig)
+        psfImageConfig = SingleFrameMeasurementConfig()
+        psfImageConfig.slots.centroid = "base_SdssCentroid"
+        psfImageConfig.slots.psfFlux = None #"base_PsfFlux"
+        psfImageConfig.slots.apFlux = "base_NaiveFlux"
+        psfImageConfig.slots.modelFlux = None
+        psfImageConfig.slots.instFlux = None
+        psfImageConfig.slots.shape = "base_SdssShape"
+        #psfImageConfig.algorithms.names = ["base_SdssCentroid", "base_PsfFlux", "base_NaiveFlux", "base_PixelFlags", "base_SdssShape"]
```

Comment:
I was experimenting with which algorithms were really needed.  I can remove this commented code, but I will replace it with a comment about what was here before.